apps/admission-webhook-k8s/admission-webhook.yaml

```diff
@@ -13,6 +13,8 @@ spec:
     metadata:
       labels:
         app: admission-webhook-k8s
+        dns-name: admission-webhook-svc
+        spiffe.io/spiffe-id-webhook: "true"
```

Comment:
Do we need this while we're using another spire label for webhook?

Reply:
When using the clustrspiffeid template you will have to somehow filter the resources so that they only apply to what is needed:  https://github.com/networkservicemesh/deployments-k8s/pull/10139/files#diff-48ae4d27db3a8aa1da0b7187a3ac5895ca91ecf2651bb6c11de6163a43f3faf3R11

Reply:
Can we remove ` spiffe.io/spiffe-id-webhook: "true"`?

Reply:
I think the only way to remove this from here is to hard code the DNS names here: https://github.com/networkservicemesh/deployments-k8s/pull/10139/files#diff-48ae4d27db3a8aa1da0b7187a3ac5895ca91ecf2651bb6c11de6163a43f3faf3R8

Reply:
This template [template](https://github.com/networkservicemesh/deployments-k8s/pull/10139/files#diff-48ae4d27db3a8aa1da0b7187a3ac5895ca91ecf2651bb6c11de6163a43f3faf3R11) should only be applied to a resource with the `spiffe.io/spiffe-id-webhook: "true"` tag, otherwise I'm not sure it will work fine

Reply:
Got the idea, removed `spiffe.io/spiffe-id-webhook` label and `app: admission-webhook-k8s` label  reused